### ISSUE TITLE
Blank squares for icons

### ISSUE THREAD

**Issue by desirajusantosh** [victoriadrake/hugo-theme-introduction#218]
![Screenshot from 2020-06-21 07-44-04](https://user-images.githubusercontent.com/3741040/85227556-13aeab00-b393-11ea-834b-a8e7f9b26574.png)
I thought to open an issue here: hugo server and hugo -D leading to icons showing up as blank squares - as seen in screenshot. 
Has anybody else faced this issue?

**Comment by victoriadrake** [comment]
Hello @desirajusantosh! The CSS will render differently if you're using the server; here's the code in question: https://github.com/victoriadrake/hugo-theme-introduction/blob/master/layouts/partials/head/css.html

If you'd like to share the HTML being rendered in your screenshot there (inspect element on one of those icons will do) I may be able to help further.

**Comment by desirajusantosh** [comment]
Thanks for quick response! This is what I see when I inspect: I earlier had "fab fab fa-twitter" but I changed that to "fab fa-twitter".
<a href="https://twitter.com/">
        <i class="fab fa-twitter"></i>
    </a>

Here's a quick link to what I have so far: https://github.com/desirajusantosh/santoshdesiraju.github.io/ Please suggest if I'm off somewhere.

**Comment by victoriadrake** [comment]
This may be related to #203. It’s necessary to set your `.icon_pack` in the site config.

I don’t see a config.toml in your repository; do you have that set up?

**Comment by desirajusantosh** [comment]
Thank you. Yes, I previously pointed you to the rendered site. But, here is the complete repo: I have the config.toml here - https://github.com/desirajusantosh/santosh/blob/master/config.toml

**Comment by victoriadrake** [comment]
Thank you for the link! I wasn't able to reproduce this issue with an identical config:

```toml
    [[params.social]]
        url   = "https://twitter.com/"
        icon  = "twitter" # icon name without the 'fa-'
        icon_pack = "fab"
```

The HTML I see rendered when using Hugo server is `<i class="fab fab fa-twitter"></i>`.

Would you ensure you're using the most current version of the theme, then let me know if this still occurs?

**Comment by desirajusantosh** [comment]
Thanks. I've retried with latest version of the theme. Its all fine until I build using hugo -D and then subsequently use github pages. That's when its failing to show icons:
Here is the latest attempt: https://github.com/desirajusantosh/me
which is showing `<i class="fab fab fa-twitter"></i>`

**Comment by victoriadrake** [comment]
I noticed you're using a few different repositories -- you may need to ensure that your `baseURL` in config.toml is correctly set.

**Comment by desirajusantosh** [comment]
Yes, config.tml has the correct baseURL. The github pages shows up but its without icons, that is all - like this: https://desirajusantosh.github.io/me/
Github pages is building from docs folder and the docs folder has been built using: HUGO_THEME=introduction hugo --gc --minify --themesDir ./themes/ -v -d ./docs

**Comment by victoriadrake** [comment]
I did notice on your site that the font is trying to load from your URL, but without the `/me/` part of the path:

![Screenshot_20200622_150022](https://user-images.githubusercontent.com/24644237/85330083-b5bcb900-b4a1-11ea-9c99-44bc21e83827.png)

However, I'm still unable to reproduce this behavior.

**Comment by mansoorbarri** [comment]
I think its because of the `integrity` tag for the `css` and `js`. I have developed multiple themes and use one for my own website as well and I have the same issue where the css won't load sometimes; removed the `integrity` tag and it works fine. Fix linked here: https://mansoorbarri.com/css-integrity-error-hugo/

**Comment by victoriadrake** [comment]
Fixed in d91c656. The root cause was a trailing slash in `$fa-font-path` combined with a leading slash in the FontAwesome `@font-face` `url()` templates, producing `webfonts//fa-*.woff2`. Most servers tolerate the double slash, but strict static hosts (S3/AWS, as a few reporters here were using) 404 it, leaving blank squares. Removed the trailing slash so the path resolves cleanly. Verified the compiled CSS now emits single-slash paths and icons render.
